src/bootstrap/step.rs

```diff
@@ -789,6 +786,10 @@ pub fn build_rules<'a>(build: &'a Build) -> Rules {
          .dep(move |s| s.name("tool-build-manifest").target(&build.config.build).stage(0))
          .run(move |_| dist::hash_and_sign(build));
 
+    rules.install("install", "path/to/nowhere")
+         .dep(|s| s.name("default:dist"))
```

Comment:
I was thinking of that too in the first place, but the install.sh generation is done as part of the dist, so it would be far less trivial.

Still doable though if you think that's a good idea.

Btw the next step could be splitting that into several rules such as install-tool-cargo or such. Just wanted to see if it was a good idea first.